challenges/arrayShift/array-shift.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+function insertShiftArray(arr, value) {
+  let middle = arr.length / 2;
+  let middleIndex = Math.ceil(middle);
+  arr.splice(middleIndex, 0, value);
```

Comment:
I would like to see this done in a way that isn't using .splice(), I would suggest using a loop here to write the logic